reo/src/run_jump_model.py

```diff
@@ -193,10 +193,6 @@ def run_jump_model(self, dfm, data, run_uuid, bau=False):
     except Exception as e:
         if isinstance(e, REoptFailedToStartError):
             raise e
-        elif "DimensionMismatch" in e.args[0]:  # bug in Xpress.jl and/or JuMP that mishandles timeouts
```

Comment:
@zolanaj it seems like we still need this block? I just ran a scenario on this branch with a 1 second timeout and the celery logs still show the `DimensionMismatch` error and the API status for results is `"status": "An error occurred. See messages for more."`